src/Routers/routes/ConsultationRoutes.tsx

```diff
@@ -22,6 +23,17 @@ const consultationRoutes: AppRoutes = {
         patientId={patientId}
       />
     ),
+  "/facility/:facilityId/patient/:patientId/encounter/:encounterId/questionnaire/:questionnaireId/responses/print":
```

Comment:
Check patient updates tab as well, we are using questionnaire responses there, so need to add route to support that.

Also check line 78, support for org/orgId access?

Reply:
Currently, facilityId is required for fetching questionnaire responses; so let's push this as first iteration and add support for print outs in org routes in a separate PR. This can be done for all printouts including prescriptions, medication administration and this.